docs/backend-requests/versioning/overview.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Versioning Overview
+description: A brief guide on how versioning works in Clerk.
+---
+
+# Versioning
+
+## Introduction
+
+When backwards incompatible changes are necessary for an API, new versions are released to prevent disruption to existing applications. These versions are identified by their release date, such as 2021-02-05. Both the frontend and backend APIs follow the same version.
+
+A complete list of all the available versions with their changes can be found on the [API versions](/docs/backend-requests/versioning/available-versions) page.
+
+## What constitutes a breaking change
+
+A breaking change is any modification that would require users to alter their existing setups. These include:
+
+- **Change in property type**: Altering the data type of a property in either requests or responses.
+- **Removal of a property**: Deleting a property or parameter from a request or response.
+- **Change in property name**: Renaming a property in request or response payloads.
+- **Backwards incompatible endpoint changes**: Implementing changes in an endpoint's functionality that older client versions cannot support. For instance, adding a new status to the sign-in process.
+- **Endpoint removal**: Discontinuing access to an existing endpoint, which remains accessible in earlier versions.
+
+## Using a specific version
```

Comment:
```suggestion
```

I suggest dropping this, and renaming the sub headings. I've suggested new names for the subheadings, and bumped them up to H2s